packages/workflows/src/handlers/inventory/remove-inventory-items.ts

```diff
@@ -20,8 +18,7 @@ export async function removeInventoryItems({
   }
 
   return await inventoryService!.deleteInventoryItem(
-    data.inventoryItems.map(({ inventoryItem }) => inventoryItem.id),
-    { transactionManager: manager }
```

Comment:
@adrien2p - and here?

Reply:
In theory, the module does not need the manager here and will rely on the compensation step if something wrong happen. Just need to check if the inventory module manages to create its own transaction internally like the new modules does. But this module has not been migrated yet

Reply:
Here you can remove. If not provided the module will start a new transaction.